### PR TITLE
Xtophs/client inner loop

### PR DESCRIPTION
**Pull request by xtophs**
# Pull Request Template

## What are you trying to address

- linter messages from recent PR

## Description of new changes

- image

## For all pull requests

- [ ] Link to the issue you are solving (so it gets closed)
- [ ] Label the pull request with the appropriate area(s)
- [ ] Assign potential reviewers (you may also want to contact them on Teams to ensure timely reviews)
- [ ] Assign the project - Engineering Playbook Backlog
- [ ] Assign the pull request to yourself

### CODE REVIEW

**Comment by xtophs** [comment]
Issue was fixed with #760

**Comment by TessFerrandez** [comment]
@xtophs the PR details suggest that the PR fixes linter issues, but it looks now to have been changed to add more content and reopened? Can you clarify? Was this by accident? Can you perhaps create a new PR for this new content with a description around what it is about. Ping me when done, I am happy to review

File: docs/developer-experience/recipes/client-app-inner-loop.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+# Strategies for a High Productivity Inner Dev Loop when developing Client Apps
```
Comment (wbreza) [inline docs/developer-experience/recipes/client-app-inner-loop.md:1]: The content looks great, but I think the title is misleading.  The same could be applied for any application component that has remote dependencies, not just client apps.

**Review by wbreza — APPROVED** [review]
Content looks great, but I would suggest renaming the page title since this pattern is not only for client app development but for any component/module that has remote dependencies.

**Comment by TessFerrandez** [comment]
I will close this as I haven't gotten a response for quite some time on this - and the PR seems to have been changed perhaps by mistake. Please re-open if you feel this is not ok.

**Pull request closed without merging**